doc/releases/changelog-dev.md

```diff
@@ -9,6 +9,9 @@
 
 <h3>Improvements</h3>
 
+* A string representation has been added to the `Molecule` class.
```

Comment:
```suggestion
* A representation has been added to the `Molecule` class.
```
bit confusing because `__str__` overrides the "string" result of a class 😄 i.e. if you explicitly `print(the_molecule)` it'd be different from just `>>> the_molecule`. Maybe you wanna change that too? I'm happy with this.